scripts/generate-docs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# This script will generate documentation for the current version
+
+VERSION=`python3 -c "import json; f = open('./package.json', 'r'); v = json.loads(f.read())['version']; f.close(); print(v)"`
```

Comment:
> Relying on python3 bin in a node package

```suggestion
VERSION=`/usr/bin/env node -e "console.log(require('./package.json').version)"`
```

Also, there should be some `if` and `exit` for if this fails.

Reply:
Hah, fair points